docsource/developers.rst

```diff
@@ -27,6 +27,134 @@ To compile, simply run ``make compile``. The
 `kb-sdk <https://github.com/kbase/kb_sdk>`_ executable must be in the system
 path.
 
+Deploying the Worspace Service locally
```

Comment:
Note that this deployment uses gridFS rather than shock as a file backend and does not support handles to shock nodes in objects, and any attempt to save an object with handles will fail.